modules/ocf_kubernetes/manifests/master.pp

```diff
@@ -62,21 +62,6 @@
       mode   => '0755';
   }
 
-  # These are needed because puppetlabs-kubernetes sets the permissions to 600
-  # but the certsign script, running under kubernetes-ca, needs to access them
-
-  File['/etc/kubernetes/pki'] {
```

Comment:
why is this being removed?

Reply:
Somehow both me and dkess wrote a fix to the kubernetes-ca permissions problem. His solution is further down and is much better than mine.

Reply:
This might still be needed, I think we added this when `puppetlabs-kubernetes` got upgraded and changed how file permissions work. Either way it should happen in a separate commit.

Reply:
They are still needed, I just tested on deadlock.